extensions/multimodal/pipelines/llava/llava.py

```diff
@@ -203,6 +203,53 @@ def placeholder_embeddings() -> torch.Tensor:
         return LLaVA_v0_Pipeline.embed_tokens(encode("<unk>"*256, add_bos_token=False)[0])
 
 
+class LLaVA_v1_5_7B_Pipeline(LLaVA_v0_13B_Pipeline):
```

Comment:
This pipeline differs from `LLaVA_v1_5_13B_Pipeline` only in terms of name and dimensions, it may be better to inherit from that instead, and remove the common methods that are not different?

Reply:
Yes that makes sense. Now `LLaVA_v1_5_7B_Pipeline` inherits from `LLaVA_v1_5_13B_Pipeline`